apigee_edge.module

```diff
@@ -1353,3 +1369,29 @@ function _apigee_edge_set_sync_in_progress(?bool $in_progress = NULL) {
 function _apigee_edge_is_sync_in_progress() {
   return _apigee_edge_set_sync_in_progress();
 }
+
+/**
+ * Gets the title of "My developer apps".
+ *
+ * @param \Drupal\user\UserInterface|null $user
+ *   It can be used if an admin visits another user's
+ *   "My developer apps" page.
+ *
+ * @return \Drupal\Core\StringTranslation\TranslatableMarkup
+ *   Title of "My developer apps".
+ */
+function apigee_edge_get_my_developer_apps_title(?UserInterface $user = NULL) {
+  $args['@developer_app'] = \Drupal::entityTypeManager()->getDefinition('developer_app')->getPluralLabel();
+  if (isset($user)) {
+    $args['@user'] = Markup::create($user->getDisplayName());
+    $title = t('@developer_app of @user', $args);
+  }
+  else {
+    $title = t('My @developer_app', $args);
+  }
+
+  // Modules and themes can alter the title.
+  \Drupal::moduleHandler()->alter('my_developer_apps_title', $title, $user);
```

Comment:
So as said in our call... `apigee_edge_my_developer_apps_title` aka. `MODULE_NAME_hook`.